### ISSUE TITLE
Change segment title 

### ISSUE THREAD

**Issue by Alkalouti** [AugustRush/ARSegmentPager#20]
Hi,

First let me thank you for this amazing third part,
but i have a problem, how i can change segment title

**Comment by AugustRush** [comment]
You can check the example and will find the protocol define 'ARSegmentControllerDelegate',just impliment '-(NSString *)segmentTitle;' method in your controller to custom the segment title.

**Comment by Alkalouti** [comment]
@AugustRush you are right but when the controller is initialized you can't change the value

**Comment by AugustRush** [comment]
ooh, i got it, but let me know why did you want to change the title? if this is a common demand，

**Comment by AugustRush** [comment]
i will add it.

**Comment by Alkalouti** [comment]
Thank you, I appreciate that so much,
i use that to change result when the data come from API and if you can add
a function for refresh main view that will make your library awesome.

Thank you again.

**Comment by AugustRush** [comment]
Okay,i got it and will add this as soon as possible.but maybe i'm little busy now.

发自我的 iPhone

> 在 2015年12月15日，16:46，Alkalouti notifications@github.com 写道：
> 
> Thank you, I appreciate that so much,
> i use that to change result when the data come from API and if you can add
> a function for refresh main view that will make your library awesome.
> 
> Thank you again.
>
